test/utils/key_events.js

```diff
@@ -0,0 +1,21 @@
+import createSyntheticEvent from 'synthetic-dom-events';
+
+export const enterEvent = createSyntheticEvent('keyup', {
+  keyCode: 13
+});
+
+export const startPointEvent = createSyntheticEvent('keydown', {
+  keyCode: 49
+});
+
+export const startLineStringEvent = createSyntheticEvent('keydown', {
+  keyCode: 50
+});
+
+export const startPolygonEvent = createSyntheticEvent('keydown', {
+  keyCode: 51
```

Comment:
Hmmm... these are part of the point, line, and polygon controls. We should fill out those docs.